.release/pypi/inference.clients.setup.py

```diff
@@ -0,0 +1,62 @@
+import os
+import shutil
+
+import setuptools
+from setuptools import find_packages
+import sys
+
+root = os.path.abspath(os.path.join(os.path.dirname(__file__), "../.."))
+sys.path.append(root)
+
+shutil.copyfile(
+    os.path.join(root, "inference/core/version.py"),
+    os.path.join(root, "inference_clients/version.py"),
+)
+
+from inference.core.version import __version__
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+
+def read_requirements(path):
+    if not isinstance(path, list):
+        path = [path]
+    requirements = []
+    for p in path:
+        with open(p) as fh:
+            requirements.extend([line.strip() for line in fh])
+    return requirements
+
+
+setuptools.setup(
+    name="inference-clients",
```

Comment:
can we change this to just `inference-client`

Reply:
👍